### PR TITLE
feat: improve layout and accessibility

### PR DESCRIPTION
**Pull request by dvalleri**
Implementations:
- improve layout
- add new version of design system
- improve accessibility

### CODE REVIEW

**Comment by devops-radicalbit** [comment]
### No vulnerabilities detected
No vulnerabilities with HIGH,CRITICAL severity have been detected in the code.

**Comment by devops-radicalbit** [comment]
### No vulnerabilities detected
No vulnerabilities with HIGH,CRITICAL severity have been detected in the code.

File: ui/src/components/JobStatus/index.jsx [hunk 1]
```diff
@@ -35,9 +39,14 @@ function JobStatus({ jobStatus }) {
...
-          description="Import a reference file to see the outcome"
-          image={<LogoSquared />}
-          title="No reference data imported yet"
+          description={(
+            <>
+              Upload a CSV file to analyze and visualize the data.
+              <br />
+              This will allow you to see the outcome of your analysis.
+            </>
+  )}
```
Comment (lucataglia) [inline ui/src/components/JobStatus/index.jsx:48]: pls format this brackets correctly

File: ui/src/components/ImportButton/import-reference-button.jsx [hunk 2]
```diff
@@ -19,16 +22,47 @@ function ImportReferenceButton({ type = 'primary-light' }) {
...
+        duration={3000}
+        force={0.9}
+        onComplete={handleOnComplete}
+        particleCount={200}
+        style={{ marginLeft: '5rem' }}
+        width={2000}
+      />
+    );
+  }
+  return false;
```
Comment (lucataglia) [inline ui/src/components/ImportButton/import-reference-button.jsx:65]: add a new line

File: ui/src/container/launchpad/alerts-list/columns.jsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  columnFactory({
+    title: 'Section',
+    key: 'anomalyType',
+    activeFilters,
+    activeSorter,
+    align: 'left',
+    width: '17%',
+    render: ({ uuid, anomalyType }) => (
+      <div className="font-[var(--coo-font-weight-bold)]">
+        <a className="pointer-events-none" href={`/models/${uuid}`}>
```
Comment (lucataglia) [inline ui/src/container/launchpad/alerts-list/columns.jsx:29]: Here you can try using a <DataTableAction /> instead of tailwind

**Review by lucataglia — CHANGES_REQUESTED** [review]

**Review by lucataglia — APPROVED** [review]

**Comment by devops-radicalbit** [comment]
### No vulnerabilities detected
No vulnerabilities with HIGH,CRITICAL severity have been detected in the code.

**Pull request merged**